bindata/cert-manager-deployment/controller/cert-manager-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Whats the reason upstream for adding this?

Reply:
Seems like this is one of the new feature additions in cert-manager 1.13, https://github.com/cert-manager/cert-manager/pull/5337 :)

cert-manager deployments would now rely upon a versioned config map implementation like revision controller in our library-go! 😮

Reply:
This will open up the operand for customisation https://github.com/AcidLeroy/cert-manager/blob/282a6d58a9038a9b54149b8f67fdc1a1a12560cc/deploy/charts/cert-manager/values.yaml#L122-L142 (which we don't want).  Since we are ensuring CM exists via the static resource controller, I'm not sure if it also forces the CM to always be empty (hoping it does), Can you verify this?

Reply:
Our staticresourcecontroller doesn't update the resource, just creation and deletion based upon specific conditions. I can confirm this because I remember that in the past we used a method of adding annotations on a service account added by staticresourcecontroller and there was noticeably no reconciles to undo them. Going through implementation suggests likewise: https://github.com/openshift/library-go/blob/master/pkg/operator/staticresourcecontroller/static_resource_controller.go

However, IMO the user changing this config map manually would open the operand for customization and we don't have a good way to restrict that yet in our current changes.